src/PrestaShopBundle/Form/Admin/Sell/Product/Description/DescriptionType.php

```diff
@@ -77,6 +77,23 @@ public function __construct(
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $productId = (int) $options['product_id'];
+        $builder
+
+            ->add('packed_products', EntitySearchInputType::class, [
+                'label' => $this->trans('Packed Products', 'Admin.Catalog.Packed'),
```

Comment:
```suggestion
                'label' => $this->trans('Pack of products', 'Admin.Catalog.Feature'),
```

Reply:
Feel free to have a look at the following documentation if you need help managing translation domains. ☺️ https://zeroheight.com/80a6c5a61/v/latest/p/738e72-manage-translation-domains

Reply:
Thx for the document ! I've just push the fix